internal/pkg/manifest/validate.go

```diff
@@ -800,6 +803,59 @@ func (r HTTPOrBool) validate() error {
 	return r.HTTP.validate()
 }
 
+func (l LoadBalancedWebServiceConfig) validateGracePeriod() error {
```

Comment:
nit: consider switching the receivers to a slightly more explicit name:
```suggestion
func (cfg *LoadBalancedWebServiceConfig) validateGracePeriod() error {
```